README.md

```diff
@@ -26,37 +26,34 @@ ONE is short for "ONE for all" and "Optimal generators with No Exception" (credi
     image.save("sd3.png")
     ```
 
-- 2024.05.23
-    1. Two OpenSora models are supported!
-        - [hpcai-OpenSora](examples/opensora_hpcai) based on VAE+STDiT
-        - [PKU-OpenSora](examples/opensora_pku) based on CausalVAE3D+Latte_T2V
-    2. [diffusers](mindone/diffusers) is now runnable with MindSpore (experimental)
-- 2024.03.22
-    1. New diffusion transformer models released!
-        - [DiT](examples/dit) for image generation
-        - [Latte](examples/latte) for video generation
-- 2024.03.04
-    1. New generative models released!
-        - [AnimateDiff](examples/animatediff) v1, v2, and v3
-        - [Pangu Draw v3](examples/pangu_draw_v3) for Chinese text-to-image generation
-        - [Stable Video Diffusion(SVD)](examples/svd) for image-to-video generation
-        - [Tune-a-Video](examples/tuneavideo) for one-shot video tuning.
-    2. Enhanced Stable Diffusion and Stable Diffusion XL with more add-ons: ControlNet, T2I-Adapter, and IP-Adapter.
-- 2023.07.01 stable diffusion 2.0 lora fine-tune example can be found [here](https://github.com/mindspore-lab/mindone/blob/master/examples/stable_diffusion_v2/lora_finetune.md)
-
-## Playground
-
-- [ChatGPT Detection](examples/detect_chatgpt): Detect whether the input texts are generated by ChatGPT
-
-- [Stable Diffusion 1.5/2.x](examples/stable_diffusion_v2): Text-to-image generation via latent diffusion models (with support for inference and finetuning)
-
-- [Stable Diffusion XL](examples/stable_diffusion_xl): New state-of-the-art SD model with double text embedders and larger UNet.
-
-- [VideoComposer](examples/videocomposer): Generate videos with prompts or reference videos via controllable video diffusion (both training and inference are supported)
-
-- [AnimateDiff](examples/animatediff): SoTA text-to-video generation models (including v1, v2, and v3) supporting motion lora fine-tuning.
-
-
-## Awesome List
-
-- [Awesome Large Vision and Foundation Models](awesome_vision.md)
+### supported models under mindone/examples
+| model  |  features  
+| :---   |  :--  |
+| [cambrian](https://github.com/mindspore-lab/mindone/blob/master/examples/cambrain)      | working on it | 
```

Comment:
multi modal understanding models and content generation models placed in different directories？